### PR TITLE
[Feat][UI] Add theme-related configuration.

### LINKED ISSUE #2135: [Feature][UI] Add theme-related configuration.
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

Add theme-related configuration.

### Usage Scenario

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by songjianet**
## Purpose of this pull request
close [#2135](https://github.com/apache/incubator-seatunnel/issues/2135)

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: seatunnel-ui/src/components/chart/index.ts [hunk 1]
```diff
@@ -16,7 +16,9 @@
  */
 
 import { getCurrentInstance, onMounted, onBeforeUnmount, watch } from 'vue'
+import { useThemeStore } from '@/store/theme'
 import { throttle } from 'echarts'
+import { useI18n } from 'vue-i18n'
 import type { Ref } from 'vue'
 import type { ECharts } from 'echarts'
```
Comment (CalvinKirs) [inline seatunnel-ui/src/components/chart/index.ts:23]: did we need add echarts license file?

Comment (songjianet) [reply seatunnel-ui/src/components/chart/index.ts:23]: I don't need it for the time being. I think it's possible to supplement the license before the release, like ds.

File: seatunnel-ui/src/components/chart/index.ts [hunk 2]
```diff
@@ -42,6 +47,22 @@ function initChart<Opt extends ECBasicOption>(
     chart && chart.resize()
   }, 20)
 
+  watch(
+    () => themeStore.darkTheme,
+    () => {
+      chart?.dispose()
+      init()
+    }
```
Comment (Amy0104) [inline seatunnel-ui/src/components/chart/index.ts:55]: Why not put the chart.dipose() into the init function?

Comment (Amy0104) [inline seatunnel-ui/src/components/chart/index.ts:55]: Why not put the chart.dipose() into the init function?

File: seatunnel-ui/src/components/chart/index.ts (hunk 1 above)
Comment (CalvinKirs) [reply seatunnel-ui/src/components/chart/index.ts:23]: I would say no. don't say like DS, we are still in the incubator, I believe that any project will strictly require a license.

**Review by Amy0104 — COMMENTED** [review]
LGTM

**Review by zhongjiajie — APPROVED** [review]

**Comment by zhongjiajie** [comment]
cc @songjianet @Amy0104 It seems currently we do not need run all ci in `ui` change anymore

**Pull request merged**